iis-wf/iis-wf-transformers/src/main/resources/eu/dnetlib/iis/wf/transformers/metadataextraction/checksum/postprocessing/meta/oozie_app/lib/scripts/transformer.pig

```diff
@@ -21,7 +21,7 @@ outputDocumentMeta = foreach joined generate
 		sourceDocumentMeta::year as year,
 		sourceDocumentMeta::publisher as publisher,
 		sourceDocumentMeta::references as references,
-		sourceDocumentMeta::authors as authors,
+		sourceDocumentMeta::authors as authorsMeta,
```

Comment:
In test for this transformer we have documentMetadata with some authors and it seems like output is correct.
AvroStorage most probably ignores field names and depends on order of fields in case when schema is defined directly in `AvroStorage('$schema_extracted_document_metadata');`